snap/implicit.go

```diff
@@ -43,6 +43,7 @@ var implicitClassicSlots = []string{
 	"opengl",
 	"network-manager",
 	"pulseaudio",
+	"modem-manager",
```

Comment:
NM, but also alternatives like connman. And, you could also manually configure modem connections.